hetznerbot/helper/hetzner.py

```diff
@@ -297,10 +307,15 @@ def format_offers(subscriber, offer_subscriber, get_all=False):
         price = offer.price / 100
         price_incl_vat = float(offer.price) * 1.19 / 100
 
+        # Get cpu data
+        cpu = session.get(Cpu, offer.cpu)
```

Comment:
This might fail if the offer got a new CPU that isn't in our set yet.

-> This should probably be optional, otherwise everything crashes until somebody adds the missing CPU.

Reply:
done